app/src/main/java/chat/rocket/android/chatdetails/ui/ChatDetailsFragment.kt

```diff
@@ -121,35 +134,46 @@ class ChatDetailsFragment : Fragment(), ChatDetailsView {
 
     private fun addOptions(adapter: ChatDetailsAdapter?) {
         adapter?.let {
-            if (!disableMenu) {
```

Comment:
do lines 124-126 need to be captured in an else for non-widechat?

Reply:
If I understand you correctly, they already are captured in "if (!Constants.WIDECHAT)"  That's a "not", so covers the non-widechat case.

Reply:
oh oops. right.